data-analytics/iot-nirvana/pipeline/src/main/java/com/google/cloud/demo/iot/nirvana/pipeline/TemperaturePipelineOptions.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.demo.iot.nirvana.pipeline;
+
+import org.apache.beam.sdk.options.Description;
+import org.apache.beam.sdk.options.PipelineOptions;
+
+/**
+ * Configuration options of the {@link TemperaturePipeline} pipeline.
+ */
+public interface TemperaturePipelineOptions extends PipelineOptions {
```

Comment:
extends PipelineOptions, GcpOptions ? This way you can get the GCP project needed for DatastoreIO.

Reply:
Done in commit ddea51860becf7a3fa4464cc9614fb868453d72a